samples/GetAddressService/Pages/Addresses.al

```diff
@@ -14,16 +14,28 @@ page 50103 Addresses
     ShowFilter = False;
     LinksAllowed = False;
 
+    Caption = 'Addresses';
+
     layout
     {
         area(Content)
         {
             repeater(General)
             {
-                field(Address;Address) { ApplicationArea = All; }
-                field(Locality;Locality) { ApplicationArea = All; }
-                field("Town/City";"Town/City") { ApplicationArea = All; }
-                field(County;County) { ApplicationArea = All; }
+                Caption = 'General';
+
+                field(Address;Address) { 
```

Comment:
The formatting style that we would like is:
```
field(Address;Address)
{
    Caption = ...
    ApplicationArea = ...
}
```

Reply:
I just turned this into a commit [4aa9cd0e](https://github.com/Microsoft/AL/pull/968/commits/4aa9cd0eed7df2f32f9439fc458ed6ae3d32627d)